light/example_test.go

```diff
@@ -11,6 +11,7 @@ import (
 	dbm "github.com/tendermint/tm-db"
 
 	"github.com/tendermint/tendermint/abci/example/kvstore"
+	"github.com/tendermint/tendermint/internal/test"
```

Comment:
The linter seems to be complaining about this.

Reply:
Fixed 👍